code/modules/mob/living/carbon/carbon.dm

```diff
@@ -205,7 +205,8 @@ CREATION_TEST_IGNORE_SELF(/mob/living/carbon)
 
 /mob/living/carbon/on_fall()
 	. = ..()
-	loc.handle_fall(src)//it's loc so it doesn't call the mob's handle_fall which does nothing
```

Comment:
loc?.handle_fall(src)

Reply:
I see this first, what does ? exactly do?

Reply:
Aight I did it that way, thanks for enlightening me